addons/account/migrations/13.0.1.1/end-migration.py

```diff
@@ -11,7 +11,21 @@ def add_default_account_data(env):
     Need to be on end-migration for making sure all chart of templates have
     their data loaded.
     """
+    def fill_chart_defaults(chart_tmpl):
```

Comment:
Please get info recursively in a similar way of core.

Reply:
what do you mean? is it's already a recursive method

Reply:
And why isolate it for this and not for the rest of values? You are also ignoring direct values in first CoA. You are forcing them to have a parent.

Reply:
which other values? I am not forcing anything, see `if not chart_tmpl[key] and chart_tmpl.parent_id[key]:` condition.

Reply:
And all under `if chart_tmpl.parent_id:`, so not having parent, it will never reach that code. And again: why not putting everything together in the same flow for not repeating things? DRY principle.

Reply:
Well, at this moment my only interest is in making green the CI. Later, once CI is green, I can try to do a more dry code.